magit-todos.el

```diff
@@ -359,6 +359,11 @@ from the \"topic2\" branch, this option could be set to
 \"topic\"."
   :type 'string)
 
+(defcustom magit-todos-show-submodule-notes t
+  "Show git submodule todo notes."
+  :type 'boolean
+  :version "1.5.2")
```

Comment:
According to the Elisp manual section 15.1:

```
‘:version VERSION’
     This keyword specifies that the item was first introduced in Emacs
     version VERSION, or that its default value was changed in that
     version.  The value VERSION must be a string.

‘:package-version '(PACKAGE . VERSION)’
     This keyword specifies that the item was first introduced in
     PACKAGE version VERSION, or that its meaning or default value was
     changed in that version.  This keyword takes priority over
     ‘:version’.

     PACKAGE should be the official name of the package, as a symbol
     (e.g., ‘MH-E’).  VERSION should be a string.  If the package
     PACKAGE is released as part of Emacs, PACKAGE and VERSION should
     appear in the value of ‘customize-package-emacs-version-alist’.
```
So:

1.  The version 1.5.2 has already been released, so that would be the wrong number.
2.  The correct keyword would be `:package-version`.

However, I don't think it's necessary to specify this anyway.